### PR TITLE
TASK: Require v7 of neos/neos-ui

### PR DESCRIPTION
**Pull request by kdambekalns**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "neos/neos-development-collection": "7.0.x-dev",
     "neos/flow-development-collection": "7.0.x-dev",
     "neos/demo": "@dev",
-    "neos/neos-ui": "@dev",
+    "neos/neos-ui": "^7.0@dev",
```
Comment (albe) [inline composer.json:22]: Do we maybe need to handle that package in the set-dependencies script?

Comment (kdambekalns) [reply composer.json:22]: Sounds like it… probably should have done that when we first decided to keep branches in sync for the UI package(s)…

Comment (albe) [reply composer.json:22]:
```suggestion
    "neos/neos-ui": "7.0.x-dev",
```
then maybe to be in line with the others?

**Review by albe — APPROVED** [review]
Okay to do this now. We need to check if/how to automate this with the `set-dependencies` script, because the releases are not automatically done together, so that could lead to requiring non-existing neos-ui versions (in theory)

File: composer.json (hunk 1 above)
Comment (kdambekalns) [reply composer.json:22]: Well, it's not the same repo and I am not sure it's "version-synced". E.g. there is only UI 4.0.x for all of Neos 4.x. Then again, for v5 we have "all branches".

**Pull request merged**